src/main/java/com/github/princesslana/slothbot/Config.java

```diff
@@ -20,6 +21,9 @@
   private static final Limiter LIMITER =
       new Limiter(SMALLD, MESSAGE_COUNTER, EXECUTOR, Path.of(getDataPath(), "limits.json"));
   private static final Self SELF = new Self(SMALLD);
+  private static final Moderator MODERATOR =
+      new Moderator(Path.of(getDataPath(), "moderator.json"));
+  private static final Set<String> SENSITIVE_COMMANDS = Set.of("buckets", "ratelimit");
```

Comment:
Since there's not really any public commands, I think I'd prefer to whitelist public commands. I guess this would mean the middleware has to check for Admin permission as well tho?